content/en/agent/kubernetes/helm.md

```diff
@@ -107,27 +62,42 @@ tiller-deploy-f54b67464-jl5gm 1/1 Running 0 3h16m
 
 ## Installing the Datadog Helm chart
 
+**Note** We recommend using Helm 3.0 or greater. If you're using a lower version, you may need to install the Helm Server, [Tiller][11].
+
+### Standard Install
 To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datadog API key from your [Agent installation instructions][4] and run:
 
 {{< tabs >}}
+{{% tab "Helm v3+" %}}
+```bash 
+helm install <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
+```
+{{% /tab %}}
 {{% tab "Helm v1/v2" %}}
-
-```bash
+```shell
 helm install --name <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
 ```
-
 {{% /tab %}}
+{{< /tabs >}}
 
-{{% tab "Helm v3+" %}}
+This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet. It also optionally deploys the [kube-state-metrics chart][5] and uses it as an additional source of metrics about the cluster. A few minutes after installation, Datadog begins to report hosts and metrics.
 
-```bash
-helm install <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
+### Standard Install with APM and DogStatsD Enabled
+For a quick setup that will automatically enable APM and\or Custom DogStatsD metrics, you can use the following command:
+{{< tabs >}}
+{{% tab "Helm v3+" %}}
+```bash 
+helm install <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY>,datadog.apm.enabled=true,datadog.statsd.enabled=true,datadog.clusterAgent.admissionController.enabled=true stable/datadog
```

Comment:
Can the admissionController be enabled by default without any other change? I feel like this should be a separate PR to make it incremental. This feels too big a change.